project/Dependencies.scala

```diff
@@ -57,7 +57,7 @@ object Dependencies {
   private val guavaV = "31.0.1-jre"
   private val heterodonV = "1.0.0-beta3"
   private val hsqldbV = "2.6.1"
-  private val http4sV = "0.21.31" // this release is EOL. We need to upgrade further for cats3. https://http4s.org/versions/
+  private val http4sV = "0.21.33"
```

Comment:
I don't know how Scala Steward selects what version to update to but [`0.21.33` is just as EOL](https://http4s.org/versions.html#which-version-is-right-for-me) as `0.21.31` was.

I would suggest that we either upgrade to the latest stable version, or leave the comment in place.

Reply:
Good catch! I'll try stable quickly now, and if it still seems to have issues I'll fallback to replacing the comment.